ndm-exporter/collector/seachest.go

```diff
@@ -185,8 +211,30 @@ func (sc *SeachestMetricData) getSeachestData() error {
 		klog.Errorf("error fetching basic disk info using seachest. %s", seachest.SeachestErrors(err))
 		return fmt.Errorf("error getting seachest data for metrics. %s", seachest.SeachestErrors(err))
 	}
+
 	sc.TempInfo.TemperatureDataValid = sc.SeachestIdentifier.GetTemperatureDataValidStatus(driveInfo)
 	sc.TempInfo.CurrentTemperature = sc.SeachestIdentifier.GetCurrentTemperature(driveInfo)
+	sc.TempInfo.LowestTemperature = sc.SeachestIdentifier.GetLowestTemperature(driveInfo)
+	sc.TempInfo.HighestTemperature = sc.SeachestIdentifier.GetHighestTemperature(driveInfo)
```

Comment:
The temperature fetching code is commented out in seachest probe. May need to uncomment it.

Reply:
@akhilerm Just checked, it's not commented.

Reply:
These are the commented out code
1. Rotation rate related fields [here](https://github.com/openebs/node-disk-manager/blob/d866a80652c63c1cba5cc354137caeaedb3418f7/cmd/ndm_daemonset/probe/seachestprobe.go#L125)
2. Temperature Data related fields [here](https://github.com/openebs/node-disk-manager/blob/d866a80652c63c1cba5cc354137caeaedb3418f7/cmd/ndm_daemonset/probe/seachestprobe.go#L164)

You need to remove that also, then only the metrics will be fetched.

Reply:
The code that is commented is responsible for filling in the block device struct of NDM.  We're fetching the details from seachest package by giving the block device path and exporting those details with the prometheus exporter, it should work without uncommenting too but was there a reason it was commented in the first place(guessing performance issues and waiting for periodic fetching thing)?

Reply:
ohkay. got it. Thanks.

> was there a reason it was commented in the first place

It was removed because the blockdevice struct did not have those fields. Earlier it was used when the disk resource was available. When moved to blockdevice resource all fields were not directly ported. Fields that were useful only were added as the need arises.

Reply:
Oh ok. We can populate the block device struct with seachest data but is there a use case for that?

Reply:
Yes, The blockdevice struct can be populated. The use case is, we need it for fetching the metrics.